multimodal/tests/unittests/test_backward_compatibility.py

```diff
@@ -6,7 +6,7 @@
 
 from autogluon.multimodal import MultiModalPredictor, AutoMMPredictor
 from autogluon.multimodal.utils import download
-from datasets import AmazonReviewSentimentCrossLingualDataset
+from unittest_datasets import AmazonReviewSentimentCrossLingualDataset
```

Comment:
Does the name `datasets` cause confusion? Since it is can only be used in in this unit tests folder, I feel the previous name should be fine.

Reply:
Yes, it conflicts with the datasets packagez